Player.gd

```diff
@@ -15,7 +15,7 @@ func _physics_process(delta: float) -> void:
 		velocity = velocity.move_toward(direction * max_speed, acceleration * delta)
 	
 	if Input.is_action_just_released("flip_maneuver"):
```

Comment:
Feels a bit weird that all the other movement keys I press and hold but the flip has to be released before it takes effect. I think it would be more intuitive if we used `is_action_pressed` instead.

Reply:
What happens when velocity is zero? 

With the current code, when velocity is zero it flips to the opposite side of the ship's nose because of the +180, which I thought was the most intuitive result. If you use is_action_pressed and velocity is zero, the ship my flip a few times before you release S despite your intended number of flips. One solution for this is to multiple velocity by -1 instead of adding 180 at the end. This solves the multiple accidental flips but it also makes it so that nothing happens when you press S and velocity is zero. What do you think? What options is best?